### PR TITLE
Update templates to match latest implementation

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@miltontony ready for review :) :dancers:

File: molo/profiles/templates/profiles/editprofile.html [hunk 1]
```diff
@@ -2,15 +2,16 @@
...
-	{% csrf_token %}
-	  {% trans "Edit your Profile" %}
-	  <table border="0">
-	            {{ form.as_table }}
-	  </table>
-	  <input type="submit" value='{% trans "Update profile" %}' class="btn btn-wide">
-	</form>
+<div class="signin-form">
+	<div class="block turq">
+		<form method="post" id="frmProfileEdit" action="{% url 'molo.profiles:edit_my_profile' %}">{% csrf_token %}
```
Comment (miltontony) [inline molo/profiles/templates/profiles/editprofile.html:7]: This form (and TuneMe) is missing the `date of birth` field, meaning a user can never change it, which is bad..

File: molo/profiles/templates/profiles/viewprofile.html [hunk 2]
```diff
@@ -3,20 +3,14 @@
 
 {% block content %}
 
-<div class="profile-style clearfix">
-  <h4>{{ username }}</h4>
-  <label class="label-wide">{% trans "Display Name:" %}
-    {{ alias }}
-  </label>
-  <label class="label-wide">{% trans "Date of Birth:" %}
-    {{ date_of_birth | date:"M d, Y" }}
```
Comment (miltontony) [inline molo/profiles/templates/profiles/viewprofile.html:12]: Why are we not showing the DOB anymore?

**Comment by smarzban** [comment]
@miltontony ready for review :)

**Comment by miltontony** [comment]
not happy with this approach since it introduces 2 new templates which will make it tricky upgrading existing apps. Lets give it another go today and see if we can't make it work.

**Comment by smarzban** [comment]
@miltontony ready for review sir :dancers: :dancers:

**Comment by miltontony** [comment]
:+1: :cake: :bouquet:

**Pull request merged**